esp/logging/loggingmanager/loggingmanager.cpp

```diff
@@ -291,36 +291,39 @@ bool CLoggingManager::saveToTankFile(IEspUpdateLogRequestWrap& logRequest, CLogR
 
 unsigned CLoggingManager::serializeLogRequestContent(IEspUpdateLogRequestWrap* request, const char* GUID, StringBuffer& logData)
 {
-    addXMLItemToBuf(LOGREQUEST_GUID, GUID, logData);
+    addXMLItemToBuf(LOGREQUEST_GUID, GUID, false, logData);
```

Comment:
Haven't looked, so probably right, but why does this not need to be encoded?

Reply:
@afishbeck GUID is a random number + data/time. Does it need to be encoded?